apps/ui/src/components/ProposalResults.vue

```diff
@@ -71,70 +71,50 @@ const results = computed(() =>
     </div>
   </div>
   <template v-else>
-    <div v-if="proposal.type !== 'basic'" class="flex flex-col gap-2">
+    <div v-if="withDetails" class="flex flex-col gap-2">
       <div
         v-for="(choice, id) in proposal.choices"
         :key="id"
-        class="flex justify-between border rounded-lg px-3 py-2.5 last:mb-0 text-skin-link relative overflow-hidden"
+        class="flex gap-2 border rounded-lg px-3 py-2.5 last:mb-0 text-skin-link relative overflow-hidden items-center"
       >
-        <div class="truncate mr-2 z-10" v-text="choice" />
-        <div class="z-10" v-text="_p(proposal.scores[id] / (proposal.scores_total || Infinity))" />
         <div
-          class="absolute bg-skin-border top-0 bottom-0 left-0 pointer-events-none"
+          class="absolute bg-skin-border top-0 bottom-0 left-0 pointer-events-none -z-10"
+          :class="{ [`_${id + 1} choice-bg opacity-20`]: proposal.type === 'basic' }"
```

Comment:
Looks good, can you just use opacity-10 instead and use the choice color as border, like this:
<img width="385" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/16245250/f7e98ac2-4baf-4624-9e3c-0eecbb908a8d">